var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -30,12 +30,23 @@ class Nwchem(Package):
 
     variant("openmp", default=False, description="Enables OpenMP support")
     variant("mpipr", default=False, description="Enables ARMCI with progress rank")
+    variant("fftw3", default=False, description="Link against the FFTW library")
```

Comment:
This variant seems not to be used?

Reply:
Have a look at
https://github.com/edoapra/spack/blob/6cbd58bd553c9a99593a80ba7899c2de228b5cba/var/spack/repos/builtin/packages/nwchem/package.py#L116-L119
```
        if "+fftw3" in spec:
            args.extend(["USE_FFTW3=y"])
            args.extend(["LIBFFTW3=%s" % fftw.ld_flags])
            args.extend(["FFTW3_INCLUDE={0}".format(spec["fftw-api"].prefix.include)])
```

Reply:
Fair, I was only looking at the diff. I guess that means those lines where not hit before.